recipes/fatiando/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "fatiando" %}
+{% set version = "0.4" %}
+{% set sha256 = "7709a4f7e40fd94bce8f0825d1d33094d271052374707bf2595a2cbd314401c7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: True  # [py3k]
+
+requirements:
+  build:
+    - python
+    - numpy 1.9
+  run:
+    - python
+    - future
+    - numpy 1.9
+    - scipy
+    - matplotlib
+    - numba
+    - pillow
+    - mayavi
```

Comment:
Not that if one of these dependencies does not have a version with `numpy 1.9.*` we will get an unreasonable dependency problem.